includes/manage.form.inc

```diff
@@ -443,8 +443,12 @@ function islandora_compound_object_add_parent($objects, $parent_pids) {
     foreach ($objects as $object) {
       $children = islandora_compound_object_get_parts($parent_pid, TRUE);
       $insert_seq = islandora_compound_object_get_insert_sequence_number($children);
+
+      $object->relationships->autoCommit = FALSE;
       $object->relationships->add(FEDORA_RELS_EXT_URI, $rels_predicate, $parent_pid);
       $object->relationships->add(ISLANDORA_RELS_EXT_URI, "isSequenceNumberOf$escaped_pid", $insert_seq, RELS_TYPE_PLAIN_LITERAL);
+      $object->relationships->commitRelationships();
+      $object->relationships->autoCommit = TRUE;
```

Comment:
By default, the 'commitRelationships()` call with flip this back, so it isn't necessary to do so separately.